eng/pipelines/templates/steps/publish-blobs.yml

```diff
@@ -0,0 +1,23 @@
+parameters:
+  FolderForUpload: ''
+  BlobSASKey: ''
+  TargetLanguage: ''
+  BlobName: ''
+  ScriptPath: ''
+
+steps:
+- pwsh: |
+    Invoke-WebRequest -MaximumRetryCount 10 -Uri "https://aka.ms/downloadazcopy-v10-windows" `
+    -OutFile "azcopy.zip" | Wait-Process; Expand-Archive -Path "azcopy.zip" -DestinationPath "$(Build.BinariesDirectory)/azcopy/"
+  workingDirectory: $(Build.BinariesDirectory)
+  displayName: Download and Extract azcopy Zip
+
+- task: Powershell@2
```

Comment:
We should understand if we can use the built-in azure blob devops task to upload these now instead of manually downloading azcopy and using it.

Reply:
I created this [issue](https://github.com/Azure/azure-sdk-tools/issues/435) to track that.